agent/handlers/types/v1/response.go

```diff
@@ -40,4 +40,12 @@ type ContainerResponse struct {
 	DockerId   string
 	DockerName string
 	Name       string
+	Networks   []NetworkResponse
+}
+
+// NetworkResponse is the schema for the network response JSON object
+type NetworkResponse struct {
```

Comment:
You should just reuse this https://github.com/aws/amazon-ecs-agent/blob/cc9fcff8e6977895cfafc49e04bdb4df3945b55a/agent/containermetadata/types.go#L86. I don't think you need to create a new struct